paramak/parametric_neutronics/neutronics_model_from_reactor.py

```diff
@@ -101,11 +101,9 @@ def __init__(
 
         # Only 360 degree models are supported for now as reflecting surfaces
         # are needed for sector models and they are not currently supported
-        if reactor.rotation_angle != 360:
-            reactor.rotation_angle = 360
-            print('remaking reactor as it was not set to 360 degrees')
-            reactor.solid
-            # TODO make use of reactor.create_solids() here
+        if hasattr(reactor, 'rotation_angle'):
+            if reactor.rotation_angle != 360:
+                reactor.rotation_angle = 360
```

Comment:
```suggestion
            reactor.rotation_angle = 360
```
This should simplify stuff + raise the diff coverage up to 100%

Reply:
ah yes, no need to check. It can just be set.